Cauldron/Assets/Scripts/CatTips/TipsCallers/SpecialVisitorsTips.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Collections;
+using CauldronCodebase.GameStates;
+using Save;
+using UnityEngine;
+using Zenject;
+
+namespace CauldronCodebase.CatTips
+{
+    public class SpecialVisitorsTips : TipsCaller, IDataPersistence
+    {
+        [Inject] private DataPersistenceManager dataPersistenceManager;
+        
+        private bool DarkStrangerCame, WitchCame, InquisitorCame;
+
+        protected override void Start()
+        {
+            dataPersistenceManager.AddToDataPersistenceObjList(this);
+            
+            base.Start();
+        }
+
+        protected override void CallTips(GameStateMachine.GamePhase gamePhase)
+        {
+            base.CallTips(gamePhase);
+
+            StartCoroutine(CheckSpecialVisitors());
+        }
+
+        private IEnumerator CheckSpecialVisitors()
+        {
+            yield return new WaitForSeconds(settings.catTips.VisitorCheckDelay);
+            
+            switch (gameDataHandler.currentCard.villager.name)
+            {
+                case "Inquisition":
```

Comment:
Let's create a static helper for these string consts (EncounterIdents?), and make the HashSet<string> GetAllSpecialCharacters() static method there. A static class with consts will help to get these strings in one place. Also check VisitorManager for them and other classes.